MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -125,11 +125,9 @@ void Setup()
 			var vp = gd.Viewport;
 
 		    Matrix projection;
-#if PSM || DIRECTX
             Matrix.CreateOrthographicOffCenter(0, vp.Width, vp.Height, 0, -1, 0, out projection);
```

Comment:
(and actually the `Draw_with_SpriteSortMode` tests would have told us that, if the OpenGL test suite was up and running... but there are so many failing tests in there right now, that I didn't spot that one.)